pkg/kv/kvserver/stores_server.go

```diff
@@ -63,7 +64,7 @@ func (is Server) CollectChecksum(
 				// snapshot (if present) intact.
 				if len(req.Checksum) > 0 {
 					log.Errorf(ctx, "consistency check failed on range r%d: expected checksum %x, got %x",
-						req.RangeID, req.Checksum, ccr.Checksum)
+						req.RangeID, redact.Safe(req.Checksum), redact.Safe(ccr.Checksum))
```

Comment:
see question above about numeric values

Reply:
These checksums are byte slices, so these definitely need the `redact.Safe`.